pubspec.yaml

```diff
@@ -0,0 +1,219 @@
+  args: 2.7.0
+  async: 2.13.0
+  boolean_selector: 2.1.2
+  characters: 1.4.0
+  checked_yaml: 2.0.3
+  cli_config: 0.2.0
+  clock: 1.1.2
+  code_assets: 0.19.0
+  collection: 1.19.1
+  convert: 3.1.2
+  coverage: 1.13.1
+  crypto: 3.0.6
+  csslib: 1.0.2
+  cupertino_icons: 1.0.8
+  device_info: 2.0.3
+  device_info_platform_interface: 2.0.1
+  fake_async: 1.3.3
+  ffi: 2.1.4
+  file: 7.0.1
+  fixnum: 1.1.1
+  flutter_gallery_assets: 1.0.2
+  flutter_localized_locales: 2.0.5
+  flutter_staggered_grid_view: 0.7.0
+  frontend_server_client: 4.0.0
+  gcloud: 0.8.18
+  glob: 2.1.3
+  google_fonts: 6.2.1
+  google_identity_services_web: 0.3.3+1
+  google_mobile_ads: 5.1.0
+  googleapis: 12.0.0
+  googleapis_auth: 1.6.0
+  hooks: 0.19.1
+  html: 0.15.6
+  http: 1.4.0
+  http_multi_server: 3.2.2
+  http_parser: 4.1.2
+  intl: 0.20.2
+  io: 1.0.5
+  isolate: 2.1.1
+  js: 0.7.2
+  json_annotation: 4.9.0
+  leak_tracker: 11.0.1
+  leak_tracker_flutter_testing: 3.0.10
+  leak_tracker_testing: 3.0.2
+  logging: 1.3.0
+  matcher: 0.12.17
+  material_color_utilities: 0.11.1
+  meta: 1.16.0
+  metrics_center: 1.0.13
+  mime: 2.0.0
+  native_toolchain_c: 0.16.1
+  nested: 1.0.0
+  node_preamble: 2.0.2
+  package_config: 2.2.0
+  path: 1.9.1
+  path_provider: 2.1.5
+  path_provider_android: 2.2.17
+  path_provider_foundation: 2.4.1
+  path_provider_linux: 2.2.1
+  path_provider_platform_interface: 2.1.2
+  path_provider_windows: 2.3.0
+  petitparser: 6.1.0
+  platform: 3.1.6
+  plugin_platform_interface: 2.1.8
+  pool: 1.5.1
+  process: 5.0.4
+  process_runner: 4.2.0
+  protobuf: 4.1.0
+  provider: 6.1.5
+  pub_semver: 2.2.0
+  pubspec_parse: 1.5.0
+  rally_assets: 3.0.1
+  retry: 3.1.2
+  scoped_model: 2.0.0
+  shelf: 1.4.2
+  shelf_packages_handler: 3.0.2
+  shelf_static: 1.1.3
+  shelf_web_socket: 2.0.1
+  shrine_images: 2.0.2
+  source_map_stack_trace: 2.1.2
+  source_maps: 0.10.13
+  source_span: 1.10.1
+  stack_trace: 1.12.1
+  standard_message_codec: 0.0.1+4
+  stream_channel: 2.1.4
+  string_scanner: 1.4.1
+  sync_http: 0.3.1
+  term_glyph: 1.2.2
+  test: 1.25.15
+  test_api: 0.7.4
+  test_core: 0.6.8
+  typed_data: 1.4.0
+  url_launcher: 6.3.1
+  url_launcher_android: 6.3.16
+  url_launcher_ios: 6.3.3
+  url_launcher_linux: 3.2.1
+  url_launcher_macos: 3.2.2
+  url_launcher_platform_interface: 2.3.2
+  url_launcher_web: 2.4.1
+  url_launcher_windows: 3.1.4
+  vector_math: 2.1.4
+  video_player: 2.9.5
+  video_player_android: 2.8.4
+  video_player_avfoundation: 2.7.1
+  video_player_platform_interface: 6.3.0
+  video_player_web: 2.3.5
+  vm_service: 15.0.1
+  vm_snapshot_analysis: 0.7.6
+  watcher: 1.1.1
+  web: 1.1.1
+  web_socket: 1.0.1
+  web_socket_channel: 3.0.3
+  webdriver: 3.1.0
+  webkit_inspection_protocol: 1.2.1
+  webview_flutter: 4.9.0
+  webview_flutter_android: 3.16.9
+  webview_flutter_platform_interface: 2.13.0
+  webview_flutter_wkwebview: 3.22.0
+  xdg_directories: 1.1.0
+  xml: 6.5.0
+  yaml: 3.1.3
+
+dev_dependencies:
```

Comment:
How did you decide what ends up here?

If it doesn't matter, I imagine just folding these into `dependencies` will be easier for tools and humans

Reply:
I would prefer to just have the single list if we aren't going to check in (and enforce) the tool - folks will try to figure out what to do and it will waste time.

Reply:
SG!

Reply:
This was resolved but I still see `dev_dependencies`?

Reply:
You should not - that is an older hash right? At least I don't see it...

Reply:
Ok I refreshed GH and it's gone, weird. Thanks!